src/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -292,7 +292,7 @@ public void Flags(AssemblyNameFlags flags)
         public void Flags_CurrentlyExecutingAssembly()
         {
             AssemblyName assemblyName = Helpers.ExecutingAssembly.GetName();
-            Assert.NotNull(assemblyName.Flags);
+            Assert.Equal(assemblyName.Flags, AssemblyNameFlags.None);
```

Comment:
Expected is the left argument, I believe.

```suggestion
            Assert.Equal(AssemblyNameFlags.None, assemblyName.Flags);
```